blocks/bricks/lookup.py

```diff
@@ -1,13 +1,16 @@
 """Introduces Lookup brick."""
-from blocks.bricks import Initializable
+from blocks.bricks import Initializable, Feedforward
 from blocks.bricks.base import application, lazy
 from blocks.roles import WEIGHT, add_role
 from blocks.utils import check_theano_variable, shared_floatx_nans
 
 
-class LookupTable(Initializable):
+class LookupTable(Initializable, Feedforward):
     """Encapsulates representations of a range of integers.
 
+    This brick can be used to embed integers, e.g. word indices,
+    in a vector space.
```

Comment:
[into](http://www.gingersoftware.com/english-online/spelling-book/confusing-words/in-into)